sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -397,9 +424,12 @@ private[ui] class ExecutionPagedTable(
 
     val desc = if (execution.description != null && execution.description.nonEmpty) {
       <a href={executionURL(execution.executionId)} class="description-input">
-        {execution.description}</a>
+        <span data-toggle="tooltip" data-placement="top" title="Click to navigate query details page.">
+          {execution.description}</span></a>
     } else {
-      <a href={executionURL(execution.executionId)}>{execution.executionId}</a>
+      <a href={executionURL(execution.executionId)}>
+        <span data-toggle="tooltip" data-placement="top" title="Click to navigate query details page.">
```

Comment:
nit: scalastyle( line exceeds 100 length)

Reply:
yes it's got exceeds, Please check this is ok ?
![Screenshot 2019-10-26 at 12 02 38 AM](https://user-images.githubusercontent.com/8948111/67595349-0e597c00-f784-11e9-8076-fa781752b2f4.png)